core/server/data/migrations/versions/3.18/01-add-email-preview-permissions-to-roles.js

```diff
@@ -0,0 +1,41 @@
+const utils = require('../../../schema/fixtures/utils');
+const logging = require('../../../../../shared/logging');
+
+const resource = 'email_preview';
+const _private = {};
+
+_private.getRelations = function getRelations(res) {
+    return utils.findPermissionRelationsForObject(res);
+};
```

Comment:
Not blocking but do we need this wrapper function? Seems like the base `utils` method name would be more descriptive if used directly in the `up`/`down` methods.

Reply:
@kevinansfield No particular reason to keep it, mostly a result of copying over pattern from existing permission update migrations. Have cleaned it up now as its definitely shorter and readable without it, thanks :)